lambda/src/main/scala/pricemigrationengine/handlers/NotificationHandler.scala

```diff
@@ -37,8 +37,8 @@ object NotificationHandler extends CohortHandler {
       )
       count <-
         subscriptions
-          .mapM(sendNotification(brazeCampaignName))
-          .fold(0) { (sum, count) => sum + count }
+          .mapZIO(sendNotification(brazeCampaignName))
+          .runFold(0) { (sum, count) => sum + count }
```

Comment:
Where have you found this change from? I can't see it in here: https://zio.dev/next/howto/migrate/zio-2.x-migration-guide/

Reply:
This was done by the scalafix step because `fold` has now been deprecated in favour of `runFold` for `ZStreams`.  I think this is to make differences between `ZIO` and `ZStream` ops clearer and to make `ZStream` ops more consistent.
https://zio.dev/next/datatypes/stream/zstream/#using-fold